Projekty/Projekt1/Jung_Kaczmarek_Marciniak/Milestone_3.ipynb

```diff
@@ -0,0 +1,7280 @@
+{
```

Comment:
<p>Widzę, że nie dokonaliście nigdzie podziału na zbiór treningowy i testowy. Dobrze że korzystaliście z walidacji krzyżowej, ale powinniście to robić w taki sposób że macie zbiór treningowy na którym za pomocą CV wybieracie parametry, a ostateczne wyniki są sprawdzane na zbiorze testowym. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/217/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>